projects/objects/buildings/protos/Silo.proto

```diff
@@ -0,0 +1,170 @@
+#VRML_SIM R2023b utf8
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
+# documentation url: https://webots.cloud/run?url=https://github.com/cyberbotics/webots/blob/released/projects/objects/buildings/protos/Silo.proto
+# keywords: building/farm
+# A steel grain silo.
+# template language: javascript
+
+EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023b/projects/appearances/protos/BrushedSteel.proto"
+
+PROTO Silo [
+  field SFVec3f    translation          0 0 0
+  field SFRotation rotation             0 0 1 0
+  field SFFloat    height               17
+  field SFFloat    radius               3
+  field SFString   name                 "silo"
+  field SFBool     enableBoundingObject TRUE    
+  field SFBool     locked               TRUE
```

Comment:
Is it really a good idea to lock it by default?
As a user I would like to insert it, move it and then lock it.